src/transforms/transform-pull-request.ts

```diff
@@ -157,3 +158,84 @@ const getBranches = async (gitHubInstallationClient: GitHubInstallationClient, p
 		}
 	];
 };
+
+export const transformPullRequest = (_jiraHost: string, pullRequest: pullRequestNode, reviews?: pullRequestNode["reviews"], log?: Logger) => {
```

Comment:
Same: we should not only look at "reviews", but also to "requested_reviews"

Reply:
included